infra/firebase.tf

```diff
@@ -20,3 +20,15 @@ resource "google_firebase_project" "default" {
 
   depends_on = [google_project_service.enabled]
 }
+
+resource "google_firebase_hosting_site" "client" {
+
+  # By default, a firebase site will be named "project_id". Only create a custom site if using suffixes
+  count = var.random_suffix ? 1 : 0
```

Comment:
question: Why wouldn't we want to create a site and just not use the suffix? Conditional resources add some complexity on reasoning that seems worth avoiding.

Reply:
I found that if you try and create a site named just the project ID, TF complains the resource already exists (which is true: you get the project_id.web.app for free). We would only want to create a site if we are using a suffix.

Reply:
sgtm